test/OpenTelemetry.Instrumentation.AspNetCore.Tests/RouteTests/README.net6.0.md

```diff
@@ -22,6 +22,7 @@
 | :green_heart: | RazorPages | [Static content](#razorpages-static-content) |
 | :green_heart: | MinimalApi | [Action without parameter](#minimalapi-action-without-parameter) |
 | :green_heart: | MinimalApi | [Action with parameter](#minimalapi-action-with-parameter) |
+| :green_heart: | ExceptionMiddleware | [Exception Handled by Exception Handler Middleware](#exceptionmiddleware-exception-handled-by-exception-handler-middleware) |
```

Comment:
Running the tests automatically updates this markdown file. If you run the tests agains .NET 7 and 8 then those markdown files will be updated as well and you can commit them.